code/game/turfs/turf.dm

```diff
@@ -346,3 +344,9 @@
 				O.singularity_act()
 	ChangeTurf(/turf/space)
 	return(2)
+
+/turf/proc/can_have_cabling()
+	return !density
+
+/turf/proc/can_lay_cable()
+	return can_have_cabling() & !intact
```

Comment:
I was talking about the can_lay_cable() proc.

On Fri, Mar 20, 2015 at 6:04 PM, xxalpha <notifications@github.com> wrote:

> In code/game/turfs/turf.dm
> <https://github.com/tgstation/-tg-station/pull/7875#discussion_r26856312>:
>
> > @@ -346,3 +344,9 @@
> >  				O.singularity_act()
> >  	ChangeTurf(/turf/space)
> >  	return(2)
> > +
> > +/turf/proc/can_have_cabling()
> > +	return !density
> > +
> > +/turf/proc/can_lay_cable()
> > +	return can_have_cabling() & !intact
>
> can_have_cabling() always returns 1 or 0, so that shouldn't matter but I
> can change it if you want.
>
> return !density because only non dense turfs should be able to have cables
> (walls are dense, floors are not dense), you told me this yourself.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tgstation/-tg-station/pull/7875/files#r26856312>.
>